### PR TITLE
add test to post controller

### PR DESCRIPTION
**Pull request by jdbs9514**
### Project requirements
- Add RSpec as a gem to your project.
- Create a Request spec file for all your controllers.
- Make sure that for each action you check:
    - If response status was correct.
    - If a correct template was rendered.
    - If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/request/posts_spec.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    end
+  end
+
+  describe 'GET /show' do
+    it 'returns http success' do
+      get '/users/:user_id/posts/:id'
+      expect(response).to have_http_status(:success)
+    end
+  end
+end
```
Comment (Bondok6) [inline spec/request/posts_spec.rb]: - [ ] Great job here 💯 However, according to the requirements, your file should include a test that ensures correct view template is rendered and a test that ensures the response body includes the correct placeholder text 👍

> For Example  💡
```ruby
    it 'Check if a correct template is rendered' do
      expect(response).to render_template(:show)
    end
```
```ruby
    it 'Check if the response body includes correct placeholder text' do
      expect(response.body).to include(' .... ')
    end
```

image

File: spec/request/users_spec.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    end
+  end
+
+  describe 'GET /show' do
+    it 'returns http success' do
+      get '/users/:id'
+      expect(response).to have_http_status(:success)
+    end
+  end
+end
```
Comment (Bondok6) [inline spec/request/users_spec.rb]: - [ ] The same thing here, Kindly check my comment above ⬆

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @jdbs9514 🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No Linter errors ✅
- Correct GitHub flow ✅
- Your repo is professional ✅


## Required Changes ♻️
- [ ] _Check the comments under the review._ ⬇

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @jdbs9514 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**